defs/README.md

```diff
@@ -358,6 +360,7 @@ REQ_TYPE
     systemd:
       <service name>: <service state>
```

Comment:
The sytanx mentioned here and in the example in test_ycheck.py  does not match.

    systemd:
        neutron-openvswitch-agent:
          state: enabled
          started-after: openvswitch-switch

Reply:
nice catch, fixed.